src/lib/commandBarConfigs/settingsCommandConfig.ts

```diff
@@ -49,6 +49,7 @@ const levelArgConfig = <T extends AnyStateMachine = AnyStateMachine>(
         ]
       : [{ name: 'User', value: 'user' as SettingsLevel, isCurrent: true }],
   machineActor: actor,
+  warningMessage: '',
```

Comment:
I think you could avoid the need for this if you make [this parameter optional](https://github.com/KittyCAD/modeling-app/pull/4076/files#diff-d718eecd2d14fae3f4ff21d3d9e0dd45af056ac3eb161f15a48d64c4183e5d75R193)

Reply:
good point, thanks!